src/image/sub_image.rs

```diff
@@ -16,7 +16,7 @@ use crate::{
 ///
 /// [`sub_image`]: trait.ImageDrawableExt.html#tymethod.sub_image
 /// [module-level documentation]: super#sub-images
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
I think we should also add `Clone` and `Copy` to the derives.

Reply:
I have just committed the new changes to the repository.